torch/testing/_internal/common_utils.py

```diff
@@ -1414,8 +1417,50 @@ def f_retry(*args, **kwargs):
     return deco_retry
 
 
-# Methods for matrix generation
+# Methods for matrix and tensor generation
+
 # Used in test_autograd.py and test_torch.py
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *,
+                low, high, requires_grad: bool = False) -> torch.Tensor:
+    """Returns a tensor of the specified size on the given device and dtype.
+       The tensors values are between -9 and 9, inclusive, unless low (high)
+       is not None in which case the values are between max(-9, low) and
```

Comment:
Could you explain the reason why you make this choice of using the max instead of just using the provided `low` value?

Reply:
Low could be -inf and we only want to generate values in a finite range, so we need some stopping point. if we only represent finite domains then it would make sense to actually use low, however that may run into other issues as some of our tests will fail when float values become too large due to cross-platform discrepancies.